packages/elasticsearch-plugin/docker-compose.yml

```diff
@@ -1,12 +1,19 @@
 version: "3"
 services:
   elasticsearch:
-    image: docker.elastic.co/elasticsearch/elasticsearch:7.10.2
+    image: docker.elastic.co/elasticsearch/elasticsearch:8.13.1
     container_name: elasticsearch
     environment:
+      - node.name=es
```

Comment:
If you pull latest changes from the `master` branch you should see that all the docker compose configs are now in a root-level compose file. Can you move these changes to there?